### PR TITLE
Add tooltip to hotbar index

### PR DESCRIPTION
**Pull request by jakolehm**
Tooltip helps to identify which hotbar is currently active.

### CODE REVIEW

**Review by nevalla — DISMISSED** [review]

**Review by jim-docker — COMMENTED** [review]
huh?
Screen Shot 2021-04-23 at 8 48 21 AM

File: src/renderer/components/hotbar/hotbar-menu.tsx [hunk 1]
```diff
@@ -63,7 +65,13 @@ export class HotbarMenu extends React.Component<Props> {
         <div className="HotbarSelector flex gaps auto">
           <Icon material="chevron_left" className="previous box" onClick={() => this.previous()} />
           <div className="box">
-            <Badge small label={hotbarIndex} onClick={() => this.openSelector()} />
+            <Badge id="hotbarIndex" small label={hotbarIndex} onClick={() => this.openSelector()} />
+            <Tooltip
+              targetId="hotbarIndex"
+              preferredPositions={TooltipPosition.TOP}
+            >
+              {hotbar.name}
```
Comment (jim-docker) [inline src/renderer/components/hotbar/hotbar-menu.tsx:74]: This is the name of the hotbar store?

File: src/renderer/components/hotbar/hotbar-menu.tsx [hunk 2]
```diff
@@ -43,6 +44,7 @@ export class HotbarMenu extends React.Component<Props> {
 
   render() {
     const { className } = this.props;
+    const hotbar = HotbarStore.getInstance();
     const hotbarIndex = HotbarStore.getInstance().activeHotbarIndex + 1;
```
Comment (jim-docker) [inline src/renderer/components/hotbar/hotbar-menu.tsx]: you don't need to get the instance again

File: src/renderer/components/hotbar/hotbar-menu.tsx (hunk 1 above)
Comment (jakolehm) [reply src/renderer/components/hotbar/hotbar-menu.tsx:74]: Sounds like migration gone south?

File: src/renderer/components/hotbar/hotbar-menu.tsx (hunk 2 above)
Comment (jakolehm) [reply src/renderer/components/hotbar/hotbar-menu.tsx]: Oh, that's typo (from auto-complete). Thanks for catching 👍

File: src/renderer/components/hotbar/hotbar-menu.tsx (hunk 1 above)
Comment (jakolehm) [reply src/renderer/components/hotbar/hotbar-menu.tsx:74]: It was the typo...

File: src/renderer/components/hotbar/hotbar-menu.tsx [hunk 3]
```diff
@@ -43,7 +44,9 @@ export class HotbarMenu extends React.Component<Props> {
 
   render() {
     const { className } = this.props;
-    const hotbarIndex = HotbarStore.getInstance().activeHotbarIndex + 1;
+    const hotbarStore = HotbarStore.getInstance();
+    const hotbar = hotbarStore.getActive();
+    const hotbarIndex = hotbarStore.activeHotbarIndex + 1;
```
Comment (Nokel81) [inline src/renderer/components/hotbar/hotbar-menu.tsx]: This line is confusing, maybe rename to `activeIndexDisplay`?

Comment (jakolehm) [reply src/renderer/components/hotbar/hotbar-menu.tsx]: Done.

**Review by Nokel81 — APPROVED** [review]

**Review by jim-docker — APPROVED** [review]
Out of scope for this PR but it still feels like this isn't enough. I have to hover over the index to see what hotbar I'm using. Can the hotbar name be a title over the catalog list? More like this is a "View", and the hotbar is a UI element belonging to the View, and the View title and hotbar name are the same. Or instead of View we can call it Workspace? :)

**Comment by Nokel81** [comment]
Maybe the name could be displayed just above the `< Index >` line.

**Pull request merged**